src/sx-design/src/ProductCard/ProductCard.js

```diff
@@ -45,51 +45,53 @@ type Props = {
 export default function ProductCard(props: Props): Element<'div'> {
   const [isHovered, setIsHovered] = useState(false);
 
-  const DEFAULT_HEIGHT = 250;
-  const DEFAULT_WIDTH = 250;
-
   return (
     <div
-      style={{ height: DEFAULT_HEIGHT, width: DEFAULT_WIDTH }}
-      className={styles('wrapper')}
+      className={styles('wrapper', 'aspectRatioBox')}
       onMouseOver={() => setIsHovered(true)}
```

Comment:
not related, but maybe
```suggestion
      onMouseEnter={() => setIsHovered(true)}
```